src/views/annual-report/annual-report.jsx

```diff
@@ -0,0 +1,1279 @@
+                                    </h4>
+                                    <p>
+                                        <FormattedHTMLMessage id="annualReport.supportersAllDescription" />
+                                    </p>
+                                    <p className="founding-partners-blurb">
+                                        <FormattedHTMLMessage id="annualReport.supportersFoundingDescription" />
+                                    </p>
+                                </div>
+                                <div className="supporters-level">
+                                    <h5>
+                                        <FormattedHTMLMessage id="annualReport.supportersFoundingTitle" />
+                                    </h5>
+                                    <hr />
+                                    <div className="supporters-list">
+                                        <ul className="supporters-list-side">
+                                            <li>Massachusetts Institute of Technology</li>
+                                            <li>National Science Foundation</li>
+                                        </ul>
+                                        <ul className="supporters-list-side">
+                                            <li>Siegel Family Endowment</li>
+                                        </ul>
+                                    </div>
+                                </div>
+                                <div className="supporters-level">
+                                    <h5>
+                                        <FormattedHTMLMessage id="annualReport.supportersCreativityTitle" />
+                                    </h5>
+                                    <hr />
+                                    <div className="supporters-list">
+                                        <ul className="supporters-list-side">
+                                            <li>Google</li>
+                                            <li>LEGO Foundation</li>
+                                            <li>Little Bluebridge Foundation</li>
+                                        </ul>
+                                        <ul className="supporters-list-side">
+                                            <li>Smilegate Foundation</li>
+                                            <li>TAL Education</li>
+                                            <li>Warner Media</li>
+                                        </ul>
+                                    </div>
+                                </div>
+                                <div className="supporters-level">
+                                    <h5>
+                                        <FormattedHTMLMessage id="annualReport.supportersCollaborationTitle" />
+                                    </h5>
+                                    <hr />
+                                    <div className="supporters-list">
```

Comment:
I wonder if these long lists of donors would be better in a separate JSON? Maybe in a follow-up pr after release